library/src/main/kotlin/com/connectrpc/protocols/GRPCInterceptor.kt

```diff
@@ -46,16 +47,10 @@ internal class GRPCInterceptor(
                     requestHeaders[GRPC_ACCEPT_ENCODING] = clientConfig.compressionPools()
                         .map { compressionPool -> compressionPool.name() }
                 }
-                val requestMessage = Buffer().use { buffer ->
-                    if (request.message != null) {
```

Comment:
This is nice that we don't have to do this any more (since unary requests always have to have messages).